### PR TITLE
Indicate registry file tree is an example

### PR DESCRIPTION
**Pull request by ncdc**

### CODE REVIEW

**Comment by ncdc** [comment]
@thoraxe @adellape PTAL

File: admin_guide/install/docker_registry.adoc [hunk 1]
```diff
@@ -177,7 +177,8 @@ Tag and image metadata is stored in OpenShift, but the registry owns layer and
 signature data. The registry stores data in a volume that is mounted into the
 registry container at *_/registry_*. You can
 link:../../dev_guide/executing_remote_commands.html[execute a remote command] in
-the registry container to see the contents:
+the registry container to see the contents (*note*: the listing below is just a
+sample; what you see will vary based on the contents of your registry):
```
Comment (adellape) [inline admin_guide/install/docker_registry.adoc]: Could you make it a "NOTE" box? I.e.:

```
the registry container to see the contents.

[NOTE]
====
The listing below is just a sample; what you see will vary based on the contents of your registry.
====
```

Comment (ncdc) [inline admin_guide/install/docker_registry.adoc]: Fixed, PTAL

Comment (adellape) [inline admin_guide/install/docker_registry.adoc]: Thx, LGTM

**Pull request merged**